src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -39,13 +44,15 @@ public virtual void Update(float elapsed)
 
         Vector3 middle = Hex.AxialToCartesian(new Hex(0, 0));
         var delta = middle - organellePos;
+        if (delta == new Vector3(0, 0, 0))
+            delta = defaultVisualPos;
         Vector3 exit = middle - delta;
         var membraneCoords = organelle.ParentMicrobe.Membrane.GetVectorTowardsNearestPointOfMembrane(exit.x,
             exit.z);
-
```

Comment:
Line breaks before `if`s are the norm in the code, so I'd argue against removing them.